packages/stark-ui/src/modules/slider/components/slider.component.ts

```diff
@@ -67,7 +67,7 @@ export class StarkSliderComponent extends AbstractStarkUiComponent implements Af
 	 * Event to be emitted when the slider's value(s) change.
 	 */
 	@Output()
-	public changed?: EventEmitter<number[]> = new EventEmitter();
+	public readonly changed?: EventEmitter<number[]> = new EventEmitter<number[]>();
```

Comment:
I think the `?: EventEmitter<number[]>` could be removed here since we initialize the EventEmitter all the time :blush: